misc/launcher.c

```diff
@@ -0,0 +1,335 @@
+/*  Setuptools Script Launcher for Windows
```

Comment:
Origin? License?

Reply:
Seams this is part of python setuptools https://svn.python.org/projects/sandbox/branches/setuptools-0.6/launcher.c

Reply:
@mmicko I made an equivalent commit to symbiyosys and needed to add a license.

Guess who `wget` the file verbatim into yosys and forgot to add the license/source :)

Reply:
@cliffordwolf Added the license and origin. Whoops :D!